ex_webrtc/lib/ex_webrtc_endpoint.ex

```diff
@@ -5,6 +5,7 @@ defmodule Membrane.RTC.Engine.Endpoint.ExWebRTC do
   use Membrane.Bin
 
   require Logger
+  require Membrane.Logger
```

Comment:
I used both Logger and Membrane.Logger, so that I get the benefits and extra information from Membrane.Logger, but can add proccess-wide metadata using Logger.metadata/1 . I will find out if setting metadata this way is impossible with Membrane.Logger

Reply:
Does `Logger.metadata` affect logs created with `Membrane.Logger`?

Reply:
It adds the metadata at the beginning of the log, but just that.